addons/account/account_move_line.py

```diff
@@ -1062,10 +1062,10 @@ def reconcile(self, cr, uid, ids, type='auto', writeoff_acc_id=False, writeoff_p
             writeoff_move_id = move_obj.create(cr, uid, {
                 'period_id': writeoff_period_id,
                 'journal_id': writeoff_journal_id,
-                'date':date,
+                'date': date,
                 'state': 'draft',
-                'line_id': writeoff_lines
```

Comment:
It's not recommended to refactor core odoo code unnecessary as it increases the chance of conflicts at a later stage. Can you restore these lines, squash commits and rename branch to something like 'writeoff_pass_context'?